_entries/2016-05-04-hyperlinks.md

```diff
@@ -35,16 +35,18 @@ Not this
 
 Don’t use meaningless terms such as ‘click here’, ‘read more’ or ‘useful links’.
 
-### Drop the http://
+### Drop the http://www
 
-When referencing a website, drop the `http://` prefix.
+When referencing a website, drop the `http://www` prefix. Verify the website doesn't need `www` to reach it.
```

Comment:
> Are you saying they should use the full URL even if there is an auto redirect in place, just to be safe?

Yea, essentially — if there’s no routing for a domain with the `www` prefix then omitting it from the link text (even if it’s there in the anchor URL itself) is imho wrong.